.github/workflows/integration.yml

```diff
@@ -44,3 +44,50 @@ jobs:
       with:
         nix_path: nixpkgs=channel:nixos-20.09
     - run: nix-shell third-party/nix/shell.nix --run "cargo build --release"
+
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: hecrj/setup-rust-action@v1
+      with:
+        rust-version: 'nightly-2021-06-29'
+        targets: 'wasm32-unknown-unknown'
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
+
+    - name: Docker meta
+      id: docker_meta
+      uses: crazy-max/ghaction-docker-meta@v1
+      with:
+        images: staketechnologies/astar-collator
+        tag-custom: shiden 
+        tag-sha: true # add git short SHA as Docker tag
+
+    - name: Checkout the source code
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: Check targets are installed correctly
+      run: rustup target list --installed
+
+    - name: Build optimized binary
+      run: |
+        cargo build --locked --release
+        cp target/release/astar-collator third-party/docker
```

Comment:
The binary name is outdated https://github.com/PlasmNetwork/Astar/blob/feature/docker-workflow/third-party/docker/Dockerfile#L32

Reply:
Yeah, fixed